Orange/widgets/visualize/owheatmap.py

```diff
@@ -618,6 +618,17 @@ def set_dataset(self, data=None):
                 self.Information.discrete_ignored(
                     ndisc, "s" if ndisc > 1 else "")
 
+        if data is not None and any(var.attributes.get('hidden', False)
```

Comment:
A block of code above this patch exclude discrete variables. Couldn't you just simply add condition there, so it would exclude discrete and hidden variables?